### DISCUSSION TITLE
Collaboration?

### DISCUSSION THREAD

**Discussion by HammyHavoc** [vrtmrz/obsidian-livesync#152] [Ideas]
Hi!

Is it currently possible to share a specific note (or canvas!) with another user (or ideally, several) on a single database? Would be amazing for collaboration and replace OneNote.

**Comment by vrtmrz** [comment]
Thank you for asking this! I have been waiting for the chance to introduce this.

Unfortunately, we cannot synchronise the specific area of the vault now, due to some chunks being shared between several notes in the local database. And chunks are encrypted in their own passphrase.
So we must decide which area can be synchronised and synchronise only them, to prevent leaking our personal notes. And probably, this process should be run even while one of them is off.

And, It can be realised by this [LiveSync-classroom](https://github.com/vrtmrz/livesync-classroom). 

We need one shared database (The same database server can be shared, only need each one's account and database), but it might be safe. 

It has a slight disadvantage compared to OneNote: it doesn't tell you who wrote it.
